src/com/lilithsthrone/game/character/GameCharacter.java

```diff
@@ -14071,7 +14071,7 @@ public String getVaginaRevealDescription(GameCharacter characterBeingRevealed, G
 			if(reactingPace!=SexPace.SUB_RESISTING
 					&& characterReacting.getFetishDesire(Fetish.FETISH_VAGINAL_GIVING).isNegative()) {
 				sb.append(UtilText.returnStringAtRandom(
-						"[npc.Name] tuts in disappointment as [npc2.namePos] [npc2.pussy+] is revealed, ",
+						"[npc.Name] [npc.verb(tut)] in disappointment as [npc2.namePos] [npc2.pussy+] is revealed, ",
```

Comment:
There is also a similar `tuts` that should be fixed in the penis reveal function... maybe others too?

Reply:
Actually, the more I look in this file, the more I find. It could end up being a chonker of a PR, so maybe don't worry about it right now.

Reply:
It's a good point, but I am not even sure this particular bit can be triggered with the MC as the tutting character. I tried to get this but I did not succeed so far.